tests/functional/test_matrix_service.py

```diff
@@ -138,6 +141,27 @@ def _post_matrix_service_request(self, bundle_fqids=None, bundle_fqids_url=None,
         data = json.loads(response)
         return data["request_id"]
 
+    def _post_notification(self, bundle_fqid):
+        data = {}
+        bundle_uuid = bundle_fqid.split('.')[0]
+        bundle_version = bundle_fqid.split('.')[1]
+
+        data["transaction_id"] = "test_transaction_id"
+        data["subscription_id"] = "test_subscription_id"
+        data["event_type"] = "CREATE"
+        data["match"] = {}
+        data["match"]["bundle_uuid"] = bundle_uuid
+        data["match"]["bundle_version"] = bundle_version
+
+        response = self._make_request(description="POST NOTIFICATION TO MATRIX SERVICE",
+                                      verb='POST',
+                                      url=f"{self.api_url}/dss/notification",
+                                      expected_status=200,
+                                      data=json.dumps(data),
+                                      headers=self.headers)
+        data = json.loads(response)
+        return data
```

Comment:
Do you think it would be worthwhile to validate the rows exist in Redshift?

Reply:
We're sending notifications for one the 5 bundles used in test so it should already be in redshift. Are you suggesting we send a delete notification first, make sure its deleted, then create, and then make sure its there?